### PR TITLE
Portfolio: mobile menu

### PR DESCRIPTION
**Pull request by AlexisJW**
- When the user clicks (or taps) the hamburger button, the mobile menu appears.

-  When the user clicks (or taps) the close (X) button, the mobile menu disappears.

- When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.

-  When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.

### CODE REVIEW

File: StyleSheet.css [hunk 1]
```diff
@@ -21,6 +21,62 @@ nav {
...
+
+.nav-bg {
+  background-color: #6070ffdc;
+  -webkit-backdrop-filter: blur(20px);
+  backdrop-filter: blur(20px);
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  inset: 0;
+}
```
Comment (tresorsawasawa) [inline StyleSheet.css:49]: - [ ] Kindly make sure your mobile menu design sticks as much as possible to the one from the Figma template. I suggest you improve your `blur effect`, `fonts on the text`, `spacing around and between elements`,... You can check all details from Figma to improve your design

| Figma design | Your Design |
| ------------------- | ------------------- |
| Screenshot from 2022-12-19 22-38-18 | Screenshot from 2022-12-19 23-30-05 |

File: StyleSheet.css [hunk 2]
```diff
@@ -21,6 +21,62 @@ nav {
...
+
+.nav-wrapper.open {
+  left: 0;
+}
+
+.nav-bg {
+  background-color: #6070ffdc;
+  -webkit-backdrop-filter: blur(20px);
+  backdrop-filter: blur(20px);
```
Comment (tresorsawasawa) [inline StyleSheet.css:44]: - [ ] To match the Figma design, kindly make sure one party of the mobile menu design can not be visible before clicking the `hamburger` menu. Try to update your design and stick as much as possible to the expected design.
Screenshot from 2022-12-19 22-34-52

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @AlexisJW 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi Team, 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🚀🔥🎉🎉 


## Highlights ✨✨
- Followed GitHub flow ✅
- Descriptivr PR ✅
- Great working on the menu functionality🔥🚀🎉
- Professional README ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**